src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -48,6 +48,17 @@ namespace NovelRT::Windowing {
     glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 0);
     glfwWindowHint(GLFW_CONTEXT_CREATION_API, GLFW_EGL_CONTEXT_API);
 
+    switch (initialWindowMode) {
+      case WindowMode::Borderless:
+        glfwWindowHint(GLFW_DECORATED, GLFW_FALSE);
+        break;
+      case WindowMode::Fullscreen:
+        _logger.logWarning("WindowMode::Fullscreen is not yet supported. Falling back to WindowMode::Windowed");
+      default:
+        glfwWindowHint(GLFW_DECORATED, GLFW_TRUE);
+        break;
+    }
```

Comment:
We probably should add bracing to each of the cases here. C++ isn't great at explaining some of the switch related errors you can get and they're basically all solved with bracing so.... :D